setup.py

```diff
@@ -42,10 +42,11 @@
         "insilicoseq>=1.5.4",
         "jsonschema>=4.0",
         "matplotlib>=3.0",
+        "nbformat>=5.0,<5.6",
```

Comment:
This was required for the Python 3.7 CI build to pass. MicroHapulator doesn't invoke it directly, but it must be used somewhere by Snakemake(?), so even after pinning the higher minimum Snakemake version, the `mhpl8r pipe` tests were breaking the build. Hopefully we can remove this dirty hack in the near future!